includes/Model/Rating.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Class Rating
+ *
+ * @link      https://github.com/googleforcreators/web-stories-wp
+ *
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ */
+
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Model;
+
+/**
+ * Class Rating
+ */
+class Rating {
```

Comment:
Might need to adjust the return types to match the schema

https://github.com/GoogleForCreators/web-stories-wp/pull/11250/files#diff-bad54f2ef7826324057d1449bd37884eab04ec70e335f6a9354850427a1d3102R243

Reply:
POC, I can add this typing in final version.